apps/omg_watcher/lib/omg_watcher/api/utxo.ex

```diff
@@ -42,4 +40,60 @@ defmodule OMG.Watcher.API.Utxo do
   def create_challenge(utxo) do
     ExitProcessor.create_challenge(utxo)
   end
+
+  @decorate measure_event()
+  @spec compose_utxo_exit(Utxo.Position.t()) :: {:ok, exit_t()} | {:error, :utxo_not_found}
```

Comment:
@pthomalla @pnowosie these functions here (and probably the on for IFEs) seems rather straightforward to (unit)test without engaging global state (`OMG.DB`). Why don't we follow the core-shell distinction here? We can then also increase the number of paths to exit composing tested a lot. We can even allow some lee-way and have the `if Utxo.Position.is_deposit?` in here, but the rest of the flow is easy to split.